src/Common/src/Interop/Windows/mincore/Interop.LUIDAndAttributes1.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Win32.SafeHandles;
+using System;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class mincore
+    {
+        internal unsafe partial struct InlineArray_LUID_AND_ATTRIBUTES_1
```

Comment:
I don't understand what value this type provides over using a ```fixed``` LUID_AND_ATTRIBUTES[] in the containing struct.